src/random/secure.cr

```diff
@@ -17,15 +19,19 @@ require "crystal/system/random"
 # For generating *high quality* random numbers, a pseudorandom number generator
 # (PRNG) such as `Random::PCG32` should be used instead, or preferably one with
 # more state (e.g. `xoshiro256**`). These number generators are much faster
-# than `Random::Secure`.
+# than `Random::Secure` and suitable for many use cases with lower
```

Comment:
*without* instead of *lower*.

For a slightly lower security (because of user-space instead of kernel), an user-space CSPRNG such as ISAAC or ChaCha20 *can* be used.

Yes, RNG and security is complex and full of devils in tiny details. I barely grasp them :sob:

Reply:
I wish someone with good knowledge of RNG and such details would write such documentation.